lib/openstudio/lib/openstudio_aws_wrapper.rb

```diff
@@ -279,11 +322,13 @@ def load_private_key(filename)
     @private_key = File.read(filename)
   end
 
-  def save_private_key(filename)
+  def save_private_key(directory = '.', filename = 'ec2_server_key.pem')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.